django/otto/settings.py

```diff
@@ -346,14 +348,18 @@
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
 
+# SC-10: Session Timeout
 SESSION_ENGINE = "django.contrib.sessions.backends.db"
+SESSION_COOKIE_AGE = 1800  # 30 minutes
```

Comment:
Unless there is a policy to require this to be 30 minutes, please leave it at 24 hours as this solves an accessibility requirement. Otherwise we need to display "session timeout" warnings and allow users to extend the session which is a bit of a pain. Can implement this if necessary of course.